README.md

```diff
@@ -18,60 +17,58 @@ WCAG Verification – 116 Questions  |  Cumulative list
 _Information and user interface components must be presentable to users in ways they can perceive._
 
 
-#### 1.1   Text alternatives-  Provide text alternatives for any non-text content so that it can be changed into other forms people need, such as large print, braille, speech, symbols or simpler language. </h4>
```

Comment:
Do we want to use the Oxford/serial comma throughout this doc? There are good arguments that it increases readability (and even that it assist with other accessibility concerns).